packages/contracts-bedrock/test/vendor/Initializable.t.sol

```diff
@@ -416,6 +416,9 @@ contract Initializer_Test is Bridge_Initializer {
         //       don't work properly. Remove these exclusions once the deployment script is fixed.
         excludes[4] = "src/dispute/FaultDisputeGame.sol";
         excludes[5] = "src/dispute/PermissionedDisputeGame.sol";
+        // TODO: Eventually remove this exlusion. Same reason as above dispute contracts.
```

Comment:
```suggestion
        // TODO: Eventually remove this exclusion. Same reason as above dispute contracts.
```

Reply:
Added in: https://github.com/ethereum-optimism/optimism/pull/11875/commits/3e8c9f9661d80e4eeae14df9a6a53b7abba600c3